src/main/java/application/bookstore/model/Book.java

```diff
@@ -0,0 +1,33 @@
+package application.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
```

Comment:
add this in order to avoid bugs related with use of `@Data` with `@Entity`
```suggestion
@Data
@EqualsAndHashCode(of = "id")
public class Book {
```